app/code/Magento/Directory/Setup/UpgradeData.php

```diff
@@ -149,6 +152,49 @@ private function getDataForAustralia()
         ];
     }
 
+    /**
+     * Mexican states data.
+     *
+     * @return array
+     */
+    private function getDataForMexico()
+    {
+        return [
+            'AGU' => 'Aguascalientes',
+            'BCN' => 'Baja California',
+            'BCS' => 'Baja California Sur',
+            'CAM' => 'Campeche',
+            'CHP' => 'Chiapas',
+            'CHH' => 'Chihuahua',
+            'CMX' => 'Ciudad de México',
```

Comment:
Hi @sivaschenko, "Mexico City" is in this line, in spanish is "Ciudad de México"